style_guide.md

```diff
@@ -0,0 +1,99 @@
+
+## Code Lay-out
+
+### 1.1 Indentation
+
+* Use 4 spaces per indentation level.
+```python
+def start(update: Update, context: CallbackContext) -> None:  
+    update.message.reply_text('Hello ! This is your bot')``
+```
+### 1.2 Maximum Line Length
+
+* Limit all lines to a maximum of 79 characters.
+### 1.3 Blank Lines
+* Use two blank lines between method definitions inside a class. 
+
+```python
+	def method_one(self): 
+	
+	
+	def method_two(self): 
+```
+	
+### 1.4 **Imports**
+
+-   Imports should be on separate lines.
+-   Group imports into three categories: standard library imports, related third-party imports, and local application/library-specific imports.
+```python
+import os
+import sys
+
+from telegram import Bot, Update
+from telegram.ext import CommandHandler, CallbackContext
+
+from my_bot.handlers import start_handler
+
+```
+---
+
+## Naming
+
+2.1 **Variables and Functions**
+* Use lowercase words separated by underscores.
```

Comment:
this is called cabab_case

Reply:
As a team, we decided to go with underscores (not hyphens), so I think it's snake_case, right?